resources/js/components/Home/RegisterForm.vue

```diff
@@ -0,0 +1,153 @@
+<script lang="ts" setup>
+import Checkbox from '@/components/Common/Checkbox.vue'
+function submitForm() {
+  // obsługuje przesyłanie formularza
```

Comment:
Do we really need this comments?

Reply:
I thought that can help for the easier indication of an entry point in the future. Maybe just a hallucination

Reply:
So please remove that

Reply:
You can just change in code, you don't have to make suggestions